src/Microsoft.AspNetCore.SignalR.Client.Core/HubConnection.cs

```diff
@@ -210,7 +215,7 @@ private static void Complete(ConnectionContext connection)
             }
         }
 
-        private async Task<ChannelReader<object>> StreamAsChannelAsyncCore(string methodName, Type returnType, object[] args, CancellationToken cancellationToken)
+        private async Task<ChannelReader<object>> StreamAsChannelCoreAsyncCore(string methodName, Type returnType, object[] args, CancellationToken cancellationToken)
```

Comment:
@davidfowl On a scale of 1-10, how mad are you right now? 😀

Reply:
![Alt Text](https://media.giphy.com/media/133DKEhjAa8aUE/giphy.gif)

Reply:
Leave it or want it to change?